test/REQUIRE

```diff
@@ -1,2 +1,2 @@
-DataStructures
+DataStructures 0.5.2
```

Comment:
are you using a feature or just relying on a bugfix? is the bugfix needed on all versions of julia that JSON supports, or just 0.6?